src/makeCalculator.js

```diff
@@ -5,6 +5,32 @@
  */
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(number) {
+      this.result += number;
```

Comment:
Ensure to handle division by zero in the `divide` method to avoid runtime errors. Consider throwing an error or returning a specific value when the `number` is 0.